spring-boot-docs/src/main/asciidoc/deployment.adoc

```diff
@@ -172,16 +172,6 @@ Spring Boot makes `-D` arguments available as properties accessible from a Sprin
 Tomcat, Jetty or Undertow instance which then uses it when it starts up. The `$PORT`
 environment variable is assigned to us by the Heroku PaaS.
 
-Heroku by default will use Java 1.6. This is fine as long as your Maven or Gradle build
-is set to use the same version (Maven users can use the `java.version` property). If you
```

Comment:
Instead of removing the paragraph altogether, it'd be nicer to upgrade it with an explanation how one can downgrade to Java7 (or 6) if need to be.